### DISCUSSION TITLE
Failed to adjust the speed of fans.

### DISCUSSION THREAD

**Discussion by FightingYao** [ipmitool/ipmitool#301] [Q&A] [answered]
Hello,
It print "Unable to send RAW comman (channel=0x0 netfn=0x30 lun=0x0 cmd=0x30 rsp=0xc1): Invalid command when I run "raw 0x30 0x30 0x01 0x00".
I want to ask how to adjust the speed of my fans. 
Thanks!

**Comment by adx-git** [comment, marked as answer]
This is not related to the IPMITool, your BMC does not support this command.

**Reply by FightingYao** [reply]
OK，thanks
